src/components/AddToCalendar/CalendarLinks.ts

```diff
@@ -0,0 +1,70 @@
+import type { CalendarEvent } from './types';
+import * as ics from 'ics';
+
+const generateGoogleCalendarLink = (event: CalendarEvent): string => {
+  const start = event.start;
+  const end = event.end;
+  const title = encodeURIComponent(event.title);
+  const description = encodeURIComponent(event.description);
+  const link = `https://www.google.com/calendar/render?action=TEMPLATE&text=${title}&dates=${formatDate(start)}/${formatDate(end)}&details=${description}&location=${event.location}&ctz=${event.timezone}`;
```

Comment:
we need to encode these URL parameters it's chopping off the Auction title in many cases since the example has a `&` in it.  We should probably construct these URLs using `URLSearchParams` and the `URL` object instead of just using templatized strings